us/en/skymiles/styles/styles.css

```diff
@@ -151,6 +151,8 @@ p small {
   font-family: var(--heading-font-family);
   font-size: var(--body-font-size-s);
   font-style: normal;
+  display: block;
+  margin-bottom: 1.063rem;
```

Comment:
I would round this up to 1rem, even if we do lose 1px. Simplifying the CSS help long-term maintainability